src/controllers/remove.director.check.answers.controller.ts

```diff
@@ -1,17 +1,54 @@
 import { NextFunction, Request, Response } from "express";
-import { REMOVE_DIRECTOR_PATH, REMOVE_DIRECTOR_SUBMITTED_PATH } from "../types/page.urls";
+import { ACTIVE_DIRECTORS_PATH, REMOVE_DIRECTOR_PATH, REMOVE_DIRECTOR_SUBMITTED_PATH } from "../types/page.urls";
 import { Templates } from "../types/template.paths";
 import { urlUtils } from "../utils/url";
+import { CompanyProfile } from "@companieshouse/api-sdk-node/dist/services/company-profile/types";
+import { getCompanyProfile } from "../services/company.profile.service";
+import { CompanyOfficer } from "@companieshouse/api-sdk-node/dist/services/officer-filing/types";
+import { getDirectorAndTerminationDate } from "../services/remove.directors.check.answers.service";
+import { Session } from "@companieshouse/node-session-handler";
+import { toReadableFormat, toReadableFormatMonthYear } from "../utils/date";
 
 export const get = async (req: Request, res: Response, next: NextFunction) => {
   try {
     const transactionId = urlUtils.getTransactionIdFromRequestParams(req);
     const submissionId = urlUtils.getSubmissionIdFromRequestParams(req);
-    const companyNumber = urlUtils.getCompanyNumberFromRequestParams(req);
+    const companyNumber= urlUtils.getCompanyNumberFromRequestParams(req);
+    const session: Session = req.session as Session;
+    const companyProfile: CompanyProfile = await getCompanyProfile(companyNumber);
+    const companyOfficer: CompanyOfficer = await getDirectorAndTerminationDate(session, transactionId, submissionId);
+    
+    if(companyOfficer.resignedOn === undefined){
+      throw Error("Resigned on date is missing for submissionId: " + submissionId);
+    }
+
+    var dateOfBirth = "";
+    if(companyOfficer.dateOfBirth !== undefined){
+      dateOfBirth = companyOfficer.dateOfBirth?.year;
+      if(companyOfficer.dateOfBirth?.day !== undefined){
+        dateOfBirth = toReadableFormat(companyOfficer.dateOfBirth?.year + "-" + companyOfficer.dateOfBirth?.month + "-" + companyOfficer.dateOfBirth?.day);
```

Comment:
Should this logic all be done in a format class like the class in utils/format.ts?

Reply:
Yeah that might be a good idea, we'll probably need the logic when we get to update. I'll move it in a future just in the interest of getting this to test ASAP